zeebe/snapshot/src/test/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksumTest.java

```diff
@@ -25,10 +25,12 @@
 import org.agrona.IoUtil;
 import org.awaitility.Awaitility;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
 import org.junit.jupiter.api.io.TempDir;
 
+@Tag("strace")
```

Comment:
:wrench: I would normally add a marker annotation to avoid everyone knowing the magical constant, and ease refactoring (you can easily refactor types via your IDE, not so much strings).

e.g. (a file which lives in the `zeebe/test-util` module next to, say, `@JMHTest`)

```java
package io.camunda.zeebe.test.util.junit;

import java.lang.annotation.Documented;
import java.lang.annotation.ElementType;
import java.lang.annotation.Inherited;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;

/**
 * Marker annotation indicating the following test requires strace to be installed on the machine.
 * By default, such tests are run only in CI where we can guarantee strace is installed. Of course
 * the test will also run when running it explicitly via your IDE or command line.
 *
 * @see io.camunda.zeebe.test.util.STracer
 */
@Target({ElementType.METHOD})
@Retention(RetentionPolicy.RUNTIME)
@Documented
@Inherited
@Tag("strace")
@EnabledIfEnvironmentVariable(named = "CI", matches = "true")
@Test
public @interface StraceTest {}
```

Then in the test where we need it (`SnapshotChecksumTest.java`), you replace:

```java
  @Test
  void shouldFlushOnPersist() throws Exception {
```

With

```java
  @StraceTest
  void shouldFlushOnPersist() throws Exception {
```